src/inspector/inspectorhandler.cpp

```diff
@@ -357,6 +358,19 @@ static QList<InspectorCommand> s_commands{
                      [](InspectorHandler*, const QList<QByteArray>& arguments) {
                        QJsonObject obj;
 
+                       QQuickWindow* window = qobject_cast<QQuickWindow*>(
+                           QmlEngineHolder::instance()->window());
+#ifndef MZ_WASM
```

Comment:
nit: Please add a comment here on why WASM is being skipped.

Reply:
The short answer is that this results in a crash on WASM, and this workaround only seems to be needed on systems that use multithreaded rendering engines (of which WASM is not). I will add some more comments and make the intent a little clearer.